packages/components/src/components/install-guide/install-instructions/Python.vue

```diff
@@ -1,18 +1,35 @@
 <template>
   <div>
     <section>
-      <h4>Installation via <code class="inline">pip</code></h4>
-      <v-code-snippet clipboard-text="pip install appmap" />
-    </section>
+      Manual installation is a good option if:
 
+      <ol>
+        <li>
+          Your project doesn't use <i>requirements.txt</i>, <i>Pipfile</i>, or <i>Poetry.lock</i>.
```

Comment:
I think the poetry one is usually all lowercase (i.e. `poetry.lock` rather than `Poetry.lock`)